diesel_derives/src/model.rs

```diff
@@ -110,7 +111,18 @@ impl Model {
     }
 
     pub fn table_name(&self) -> &Path {
-        self.table_name.as_ref().unwrap_or(&self.name)
+        match self.table_names.len() {
+            0 => &self.name,
+            1 => &self.table_names[0],
```

Comment:
Is there a reason for just returning the first table name and not any other table name?

Reply:
While `Insertable` calls `table_names` method, other derives would call this to check if there is only one table name